compatibility-verifier/sample-test-suite/config/ControllerConfig.properties

```diff
@@ -0,0 +1,24 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+controller.host = localhost
+controller.port = 9000
+controller.zk.str = localhost:2181
```

Comment:
The one used in BrokerConfig and ServerConfig is called `pinot.zk.server` while this one is called `controller.zk.str`. Why is it different?

Reply:
That is how the configs are accepted in the commands. Unifying them is another task, and a backward incompatible one (unless coded carefully)

Reply:
Will it be good to add a TODO there then?

Reply:
Not in this PR. It is a TODO in `StartControllerCommand`, perhaps, and I don't want to do that here. We could create an issue to  make the configs the same across all components.

Reply:
I thought there was already an issue for the config, and I was right.
https://github.com/apache/incubator-pinot/issues/5801